examples/kubernetes/kuma-demo/README.md

```diff
@@ -2,10 +2,10 @@
 
 ## Setup Environment
 
-1. Start a Kubernetes cluster with version 1.15 or higher and at least 4GB of memory
+1. Start a Kubernetes cluster with at least 4GB of memory
```

Comment:
I think it worth mentioning that we test `Kuma` on `Kubernetes` `v1.13.0 - v1.16.x`